components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -106,9 +105,14 @@ func newIAMUpgradeToV2Cmd() *cobra.Command {
 		false,
 		"Upgrade to version 2.1 with beta project authorization.")
 
-	// all flags are hidden right now
-	cmd.PersistentFlags().VisitAll(func(f *pflag.Flag) { f.Hidden = !isDevMode() })
-
+	var err error
+	if !isDevMode() {
+		err = cmd.PersistentFlags().MarkHidden("beta2.1")
+		if err != nil {
```

Comment:
[nit] How about
```go
if !isDevMode() {
	if err := cmd.PersistentFlags().MarkHidden("beta2.1"); err != nil {
		fmt.Printf("failed marking --beta2.1 hidden: %s\n", err.Error())
	}
}
```
So, this can never happen. No need to panic. In other places, we just ignore the error return,
```go
_ = cmd.PersistentFlags().MarkHidden("beta2.1")
```
I believe.

(ℹ️ the only way this could fail is if the string you're providing didn't match the flag.)

Reply:
I see it both ways in our codebase and was being a 🤓 about always checking errors. But happy to make the change!